app/views/govuk_publishing_components/components/docs/notice.yml

```diff
@@ -65,3 +65,13 @@ examples:
       description_govspeak: <p>This document is no longer current. We published a new version on 30 September 2015.</p>
       show_banner_title: true
       banner_title: "Information"
+  with_ga4_tracking_disabled:
+    description: |
+      Disables GA4 tracking on the component. Tracking is enabled by default. When tracking is enabled, a data module and data attributes is added to the parent element. See the [ga4-link-tracker documentation](https://github.com/alphagov/govuk_publishing_components/blob/main/docs/analytics-ga4/ga4-link-tracker.md) for more information.
```

Comment:
Maybe worth mentioning the disabling of tracking on govspeak if passed? That was helpful to know.